lib/bone.js

```diff
@@ -160,10 +160,26 @@ class Bone {
    */
   _validateAttributes(values = {}) {
     const { attributes } = this.constructor;
-    Object.keys(this.raw).map((valueKey) => {
+    // check changed values
+    const changes = this.changes();
+    let changedValue = {};
+    for (const key in changes) {
+      if (changes[key].length && changes[key].length === 2) {
```

Comment:
```suggestion
      if (changes[key].length === 2) {
```

`changes[key].length &&` seems unnecessary